src/xargs/mod.rs

```diff
@@ -813,11 +841,21 @@ fn do_xargs(args: &[&str]) -> Result<CommandResult, XargsError> {
         )
         .arg(
             Arg::new(options::VERBOSE)
-                .short('t')
-                .long(options::VERBOSE)
-                .help("Be verbose")
-                .action(ArgAction::SetTrue),
+            .short('t')
+            .long(options::VERBOSE)
+            .help("Be verbose")
+            .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::REPLACE)
+                .long(options::REPLACE)
+                .short('I')
```

Comment:
I think `-I` has to be an `Arg` on its own because its behavior is slightly different from `-i`: `-I` requires a value  whereas `-i` doesn't:
```
$ printf "foo" | xargs -I echo {} _ bar
xargs: {}: No such file or directory
$ printf "foo" | xargs -I {} echo {} _ bar
foo _ bar

$ printf "foo" | xargs -i echo {} _ bar
foo _ bar
```

Reply:
How do you handle this with clap? New Arg?

Reply:
Yes, a new `Arg` without long option.

Reply:
ok
will do @cakebaker 
do you mind if I do that in a new PR ?
thanks

Reply:
No, I don't mind :)